docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -1442,7 +1442,7 @@ must restart the server for that change to take effect.
 |`fhirServer/bulkdata/jobParameters/cos.api.key`|Y|Y|
 |`fhirServer/bulkdata/jobParameters/cos.srvinst.id`|Y|Y|
 |`fhirServer/bulkdata/bulkDataBatchJobIdEncryptionKey`|Y|Y|
```

Comment:
I think this row is wrong too

Reply:
this line is correct, the id is encrypted in the bulkdata operation side which is in the fhir-server.